README.md

```diff
@@ -16,7 +16,7 @@
 # Contributing
 
 Contributing to Masonite is simple:
-* (optionally) Hop on [Gitter.im](https://gitter.im/masonite-framework/Lobby) to ask any questions you need.
+* Hop on [Slack Chanell]!(http://slack.masoniteproject.com/) to ask any questions you need.
```

Comment:
theres a `!` after the brackets. It should be at the end of the parenthesis. Click "view" to see what i mean :)